src/api/utils/getResponseBody.js

```diff
@@ -1,9 +1,4 @@
-export default (response) => {
-  const isBodyEmpty = response.status === 204;
-
-  if (isBodyEmpty) {
-    return Promise.resolve();
-  }
-
-  return response.json();
+export default async (response) => {
```

Comment:
This is async but has no await inside 🤔

Reply:
It does, but it's implicit in the `response.json()`. The thing is that `response.json()` returns a promise, so for consistency i choose to return an empty Promise in the other cases, like we did before the change.

What do you think? Also including @mcousillas6 @bernabe9 on this

Reply:
It's weird using `camelizeKeys` if the `response` could be `undefined`. I would rather `await` for the `response.json()` to finish in order to camelize its keys.
if (!isBodyEmpty) return response.json();
`if (!isBodyEmpty) return humps.camelizeKeys(await response.json());`